doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -291,6 +291,10 @@ Cellular samples
 
   * Added support for native simulator platform and updated the documentation accordingly.
 
+* :ref:`nrf_provisioning_sample` sample:
+
+  * Refactored the sample to use Zephyr's Connection Manager feature.
```

Comment:
```suggestion
  * Updated the sample to use Zephyr's :ref:`zephyr:conn_mgr_docs` feature.
```
I believe this is about https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/connectivity/networking/conn_mgr/index.html